Tools/SandboxTest.ps1

```diff
@@ -165,11 +166,15 @@ foreach ($dependency in $dependencies) {
       $WebClient.DownloadFile($dependency.url, $dependency.SaveTo)
 
     } catch {
-      #Pass the exception as an inner exception
-      throw [System.Net.WebException]::new("Error downloading $($dependency.url).", $_.Exception)
+      # If the download failed, remove the item so the sandbox can fall-back to using the PowerShell module
+      Remove-Item $dependency.SaveTo -Force | Out-Null
     }
     if (-not ($dependency.hash -eq $(Get-FileHash $dependency.SaveTo).Hash)) {
-      throw [System.Activities.VersionMismatchException]::new('Dependency hash does not match the downloaded file')
+      # If the hash didn't match, remove the item so the sandbox can fall-back to using the PowerShell module
+      Write-Host -ForegroundColor Red '      Dependency hash does not match the downloaded file'
```

Comment:
```suggestion
      Write-Host -ForegroundColor Red "      Dependency hash does not match the downloaded file $(Split-Path -Path $dependency.SaveTo -Leaf)"
```

nit: we can add the file name, since it can theoretically print twice (not likely tho with the UI.Xaml Url)

Reply:
We could, but we shouldn't need to -
![image](https://github.com/microsoft/winget-pkgs/assets/12611259/6f82dc88-935e-4d62-93d2-302957fc064f)

Reply:
Ah - I wasn't paying attention to the full output. Looks correct